cmd/snap-repair/runner.go

```diff
@@ -80,6 +80,24 @@ func (r *Repair) SetStatus(status RepairStatus) {
 	r.run.SaveState()
 }
 
+// makeRepairSymlink ensures $dir/repair exists and is a symlink to
+// /usr/lib/snapd/snap-repair
+func makeRepairSymlink(dir string) (err error) {
+	// make "repair" binary available to the reapir scripts via symlink
+	// to the real snap-repair
+	if err = os.MkdirAll(dir, 0755); err != nil {
+		return err
+	}
+
+	old := filepath.Join(dirs.CoreLibExecDir, "snap-repair")
+	new := filepath.Join(dir, "repair")
```

Comment:
Optional nitpick: source/target are more natural here.

Reply:
Yeah, I was following the os.Symlink() convention from go here, happy to follow the symlink(2) name. Source/Target is slightly ambitious because target might mean "link-target" or "target-path-of-the-link-itself" (but maybe that is just me :)